modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/InnerNodeAsmGenerator.java

```diff
@@ -1578,6 +1592,16 @@ private void addInjectedNameFieldMethods(FieldDefinition injectedNameFieldDef) {
                 )).ret();
     }
 
+    private void implementInjectedValueFieldNameMethod(FieldDefinition injectedValueFieldDef) {
+        MethodDefinition method = innerNodeClassDef.declareMethod(
+                EnumSet.of(PUBLIC),
+                INJECTED_VALUE_FIELD_NAME_MTD_NAME,
+                type(String.class)
+        );
+
+        method.getBody().append(constantString(injectedValueFieldDef.getName())).retObject();
```

Comment:
You should call `getPublicName(injectedValueSchemaField)`, because configuration name might be different from field name